2019.08 release notes (2019-10-01).MD

```diff
@@ -0,0 +1,131 @@
+# 2019.08 release notes (2019-10-01)
+
+Packages released:
+-
+
+## Charts
+**Charts:** 
+
+- Updated Victory to 30.0.0 ([#2883] (https://github.com/patternfly/patternfly-react/pull/2883))
+
+- For consistency, we wanted to use React.Fragment. Replaced empty tags with React.Fragment ([#2988] (https://github.com/patternfly/patternfly-react/pull/2988))
+
+- Added missing style and responsive props to ChartVoronoiContainer ([#2977] (https://github.com/patternfly/patternfly-react/pull/2977))
+
+- Updated chart tutorial paths ([#2962] (https://github.com/patternfly/patternfly-react/pull/2962))
+
+- This should help eliminate the "missing key prop" warning seen in the browser console for ChartPie ([#2943] (https://github.com/patternfly/patternfly-react/pull/2943))
+
+- Fixed live edit caret color so it is visible ([#2899] (https://github.com/patternfly/patternfly-react/pull/2899))
+
+**Area chart:** 
+
+- Added stacked area chart demo and test ([#2896] (https://github.com/patternfly/patternfly-react/pull/2896))
+
+- Added Horizontal indicator example ([#2796] (https://github.com/patternfly/patternfly-react/pull/2796))
+
+**Sparkline:** Added line border to ChartArea and move sparkline ([#2887](https://github.com/patternfly/patternfly-react/pull/2887))
+
+**Donut chart:** Added simple fix for the donut chart tutorial link
+([#2940] (https://github.com/patternfly/patternfly-react/pull/2940))
+
+## Components
+
+**App launcher:** Made dropdown/app launcher work better with router components ([#3011] (https://github.com/patternfly/patternfly-react/pull/3011))
+
+**Button:** 
+
+- Added control variant to button ([#2983] (https://github.com/patternfly/patternfly-react/pull/2983))
+
+- Set the disable modifier when component is not button ([#2683] (https://github.com/patternfly/patternfly-react/pull/2683))
+
+**Dropdown:** Disabled button now only applies disabled attribute ([#2989] (https://github.com/patternfly/patternfly-react/pull/2989))
```

Comment:
Should this be a bullet?

Reply:
I think it is correct as it is. Does it look different from the others with one single line?